schemas/software.yml

```diff
@@ -19,7 +19,8 @@ language: "enum(
               'rust',
               'sql',
               'typescript',
-              'vuejs'
```

Comment:
this is a nitpick, but for long term sanity, could you ad c# to the list alphabetically?